### PR TITLE
fix: line chart animation fix 

### PR DESCRIPTION
**Pull request by Golodhros**
## Description
Fixes line animation when multiline
Fixes line demo brushing logic

## Motivation and Context
Fixing the line chart

## How Has This Been Tested?
Manually

## Types of changes

-   [ ] Refactor (changes the way we code something without changing its functionality)
-   [x] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

-   [ ] Latest master code has been merged into this branch
-   [ ] No commented out code (if required, place // TODO above with explanation)
-   [ ] No linting issues
-   [ ] Build is successful
-   [ ] Code follows the [API Guidelines](http://britecharts.github.io/britecharts/topics-index.html#toc5__anchor)
-   [ ] Updated the documentation
-   [ ] Added tests to cover changes
-   [ ] All new and existing tests passed

### CODE REVIEW

**Comment by Golodhros** [comment]
@rajeshmr82 take a look please!

File: src/charts/line/line.js [hunk 1]
```diff
@@ -1107,6 +1109,17 @@ export default function module() {
             : d1;
     }
 
+    /**
```
Comment (rajeshmr82) [inline src/charts/line/line.js:1112]: Shouldn't this be above findOutNearestDate?

**Review by rajeshmr82 — APPROVED** [review]
Looks good. A minor comment on the ordering of functions

File: src/charts/line/line.js (hunk 1 above)
Comment (Golodhros) [reply src/charts/line/line.js:1112]: Oops, you are right! I will change it!

Thanks!

**Pull request merged**